packages/datatrak-web/src/Routes.tsx

```diff
@@ -45,8 +52,13 @@ export const Routes = () => {
         <Route index element={<LandingPageRedirect />} />
         {/** Any views that should have the background image should go in here */}
         <Route path="/" element={<BackgroundPageLayout />}>
-          <Route path={ROUTES.LOGIN} element={<LoginRedirect />} />
-          <Route path={ROUTES.VERIFY_EMAIL} element={<VerifyEmailPage />} />
+          {/** Any auth views should go in here, as they have a layout where the form is centred in the page */}
+          <Route path="/" element={<AuthLayout />}>
+            <Route path={ROUTES.LOGIN} element={<LoginRedirect />} />
+            <Route path={ROUTES.VERIFY_EMAIL} element={<VerifyEmailPage />} />
```

Comment:
I think the VerifyEmailPage turned out really well. It's very nicely separated and easy to follow.